### PR TITLE
Update instructions for building prod

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 All TDR documentation is available [here](https://github.com/nationalarchives/tdr-dev-documentation)
 
-This project can be used to spin up a jenkins server using ECS. The ECS cluster is created using terraform and the jenkins configuration uses the [JCasC](https://jenkins.io/projects/jcasc/) plugin and the jenkins.yml file sets up the jenkins configuration.
+This project can be used to spin up a jenkins server using ECS. The ECS cluster is created using terraform and the jenkins configuration uses the [JCasC](https://jenkins.io/projects/jcasc/) plugin. We use this project to create two Jenkins instances. Integration Jenkins uses the jenkins.yml file for configuration and jenkins-prod.yml for the production Jenkins.
```
Comment (TomJKing) [inline README.md:5]: Maybe add a paragraph and use a list:

> ... plugin.
> 
> We use this project to create two Jenkins instances. For configuration:
> * Integration Jenkins uses the jenkins.yml file; and
> * Production Jenkins uses the jenkins-prod.yml

File: README.md [hunk 2]
```diff
@@ -83,7 +82,9 @@ Before doing any Jenkins deployments:
 
 [reset-builds]: https://github.com/nationalarchives/tdr-dev-documentation/blob/master/manual/reset-jenkins-builds.md
 
-### Deploy Jenkins Docker image
+### Deploy Jenkins Docker images
+
+There are two Jenkins instances now which need to be deployed but the process is similar.
```
Comment (TomJKing) [inline README.md:87]: Maybe reword this slightly: 

> There are two Jenkins instances, **integration and production,** which need to be ...

**Review by TomJKing — COMMENTED** [review]
Couple of minor comments

File: README.md (hunk 1 above)
Comment (MancunianSam) [reply README.md]: Yeah, that looks better. I've changed it.

File: README.md (hunk 2 above)
Comment (MancunianSam) [reply README.md]: Makes sense, I've changed it.

**Review by TomJKing — APPROVED** [review]
